### PR TITLE
Removes mesons and material scanners from the station

### PR DESCRIPTION
**Pull request by Alberyk**
What it says in the title.
The issue is that mesons and materials just end being used to powergame stuff, like seeing turrets in away site.

**Mesons will no longer hallucinations from the supermatter, safety glasses will do that now. They have been added to engineering vending machines and radiation closets.**

### CODE REVIEW

File: html/changelogs/alberyk-nomesons.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+author: Alberyk
+
+delete-after: True
+
+changes: 
+  - rscdel: "Removed mesons and material scanners from the station."
```
Comment (Geevies) [inline html/changelogs/alberyk-nomesons.yml:6]: add that safety goggles protect you from the SM here

**Review by Geevies — APPROVED** [review]

**Comment by alsoandanswer** [comment]
Wait, are you completely removing them from the game or just restricting it to protolathe?

**Comment by Alberyk** [comment]
They have been removed from the station, erts can still get them and they can be spawned. The issue is not only related to away sites, but any kind of hidden place on the station as well.

**Review by fernerr — APPROVED** [review]

**Comment by alsoandanswer** [comment]
Understandable, have a great day

**Pull request merged**